src/store/configureStore.js

```diff
@@ -1,11 +1,19 @@
 import { createStore, applyMiddleware, compose } from 'redux';
```

Comment:
Isn't the idea of adding redux persist, to save the location of the layout, so that it can be loaded in when the user restarts the application?